docs/application/native/guides/app-management/cion.md

```diff
@@ -0,0 +1,228 @@
+# Cion
+
+The Cion API provides functionality to communicate with the other devices.
+
+The main features of the Cion API include:
+
+- Communicating with other applications in server-client style.
+
+  You can communicate with other applications in [server-client](#server_client) style.
+
+- Communicating with other applications in group style.
+
+  You can communicate with other applications in [group](#group) style.
+
+- Supporting TIDL
+  You can generate communication code using [TIDL](#tidl).
+
+## Prerequisites
+
+1.  To use the Cion API (in [mobile](../../api/mobile/latest/group__CAPI__CION__MODULE.html), [wearable](../../api/wearable/latest/group__CAPI__CION__MODULE.html) and [iot](../../api/iot-headed/latest/group__CAPI__CION__MODULE.html) applications), the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
+
+    ```xml
+    <privileges>
+       <privilege>http://tizen.org/privilege/d2d.datasharing</privilege>
+       <privilege>http://tizen.org/privilege/internet</privilege>
+    </privileges>
+    ```
+
+2.  To use the functions of the Widget Service API, include the `<widget_service.h>` header file in your application:
+
+    ```cpp
+    #include <cion.h>
+    ```
+
+<a name="server_client"></a>
+## Communicate in server-client style
+
+Cion API provides functions to communicate with the other applications in server-client style.
+
+Cion client can connect to discovered server and send data.
```

Comment:
Global comment: 

Can we add some reference text for procedural steps?

Reply:
it's changed